server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/http/AuthenticationFilter.java

```diff
@@ -34,15 +34,15 @@
 import reactor.core.publisher.Mono;
 import reactor.netty.http.server.HttpServerRequest;
 
-public class AuthenticationReactiveFilter {
-    private static final Logger LOGGER = LoggerFactory.getLogger(AuthenticationReactiveFilter.class);
+public class AuthenticationFilter {
```

Comment:
Should we rename then as well `DefaultMailboxesReactiveProvisioner` back to `DefaultMailboxesProvisioner` ?